src/privileged/multipreffer/api.js

```diff
@@ -58,7 +58,7 @@ this.FirefoxHooks = {
     const prefs = this.variations[variationName].prefs;
     if (this.studyInfo.isFirstRun) {
       for (const name of Object.keys(prefs.setValues)) {
-        if (Preferences.isSet(name)) {
+        if (!prefs.expectNonDefaults.includes(name) && Preferences.isSet(name)) {
```

Comment:
I think I'd have done `(!prefs.expectNonDefaults || !prefs.expectNonDefaults.includes(name))` to avoid requiring this field, but maybe there's a reason you want that :)

Reply:
When I was writing this patch I found that quality-of-life improvements like the one you're suggesting very quickly increased the complexity of the boolean logic and tests, so if you don't mind I'd rather just leave it mandatory, at least for now. I'm still reeling from this morning :P